### PR TITLE
initial skeleton for NeXus App

### PR DESCRIPTION
**Pull request by sanbrock**

### CODE REVIEW

File: src/pynxtools/nomad/schema.py [hunk 1]
```diff
@@ -835,6 +836,7 @@ def init_nexus_metainfo():
     nexus_section = Section(
         validate=VALIDATE, name=__GROUPING_NAME, label=__GROUPING_NAME
     )
+    nexus_section.base_sections = [Schema.m_def]
```
Comment (sherjeelshabih) [inline src/pynxtools/nomad/schema.py:839]: Just for future reference. Could you describe how this affects the metainfo generation?

File: src/pynxtools/nomad/schema.py [hunk 2]
```diff
@@ -799,6 +799,7 @@ def __create_package_from_nxdl_directories(nexus_section: Section) -> Package:
                 SubSection(section_def=section, name=section.name)
             )
 
+    package.section_definitions.append(nexus_section)
```
Comment (sherjeelshabih) [inline src/pynxtools/nomad/schema.py:802]: Just for future reference. Could you describe how this affects the metainfo generation?

File: src/pynxtools/nomad/schema.py (hunk 1 above)
Comment (sanbrock) [reply src/pynxtools/nomad/schema.py:841]: added

File: src/pynxtools/nomad/schema.py (hunk 2 above)
Comment (sanbrock) [reply src/pynxtools/nomad/schema.py:803]: added

**Pull request merged**